### PR TITLE
revise README in English to make it more authentic

### PR DESCRIPTION
**Pull request by dinoallo**

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -1,34 +1,63 @@
 # SEALFS
-[English](https://github.com/labring/sealfs/blob/main/README.MD)|[简体中文](https://github.com/labring/sealfs/blob/main/README-ZH.MD)
+[English](https://github.com/labring/sealfs/blob/main/README.MD) | [简体
```
Comment (luanshaotong) [inline README.MD:2]: Do line break affects link?

**Comment by luanshaotong** [comment]
Very nice modification. That's exactly what we want.

File: README.MD (hunk 1 above)
Comment (dinoallo) [reply README.MD:2]: No, you can preview from [here](https://github.com/dinoallo/sealfs/blob/main/README.MD) before you merge it :)

Comment (luanshaotong) [reply README.MD:2]: Oh, it's cool. I found a space in it, can I remove it?

Comment (dinoallo) [reply README.MD:2]: If you are talking about the spaces between English and 简体中文, then yes, as long as you see fit. It's trivial.

Comment (luanshaotong) [reply README.MD:2]: Ok, I'll edit later.

**Review by luanshaotong — APPROVED** [review]
LGTM

**Pull request merged**